src/sass/base/_b-breakpoints.scss

```diff
@@ -1,9 +1,13 @@
 //===========================
 // Additional breakpoints
-// 
+//
 // Moved here from b-variables file so that we can load this partial
 // after USWDS variables and before other partials.
 //===========================
+// @media single-keyword helpers
+$small: new-breakpoint(min-width $small-screen $grid-columns-small);
+$medium: new-breakpoint(min-width $medium-screen $grid-columns-medium);
+$large: new-breakpoint(min-width $large-screen $grid-columns-large);
```

Comment:
This actually shouldn't change our breakpoints at all. Those declaration should really just reset things so that USWDS can be imported. This reason why is sort of confusing...

So, Foundation defines those variables, but doesn't actually use them - they only define them because previous version of Foundation did use them, and they wanted to be backwards-compatible for apps that are dependent on them. 

Then, the new version of USWDS defined those variables too, and passes them as an argument to a mixin called `media`.  Everything should have been fine since we were importing USWDS after Foundation, except that USWDS was initializing the variables with that `!default` directive, so Foundation's variable declaration were overriding USWDS's. This meant that when USWDS tried using that `media` mixin, an error would be thrown about a bad parameter. That's why we define them there - it's the same code as USWDS, but that that `!default` flag.

So basically it goes like this -

1. Import Foundation 
  - -> defines small, medium, large
2. Import USWDS variables 
  - -> defines small, medium, and large with the `!default` directive, so Foundation's override them.
3. Import our custom breakpoints file, which redeclares USWDS's small, medium, and large without the `!default` flag
4. Import the rest of USWDS, which will now work as expected.

Reply:
Thanks for the detailed explanation @ncksllvn.